### PR TITLE
Mission - find bugs

### PR DESCRIPTION
**Pull request by rudehn**
We are on a journey of finding out why the issue command system isn't working. Join me on this journey!

### CODE REVIEW

File: github_api/misc.py [hunk 1]
```diff
@@ -1,7 +1,5 @@
 from math import exp
 
-
-
```
Comment (PlasmaPower) [inline github_api/misc.py]: Please don't - this change is in #390 - let that fix it.

File: cron/poll_read_issue_comments.py [hunk 2]
```diff
@@ -224,12 +226,16 @@ def handle_comment(api, issue_comment):
             command = db_fields["command"]
             body = "> {command}\n\nVote failed".format(command=command)
             gh.comments.edit_comment(api, settings.URN, resp_id, body)
+    else:
+        __log.debug("comment: {comment} is not a command, " +
+                    "parsed: {parsed}".format(comment=comment_text, parsed=orig_parsed))
 
-
+
```
Comment (andrewda) [inline cron/poll_read_issue_comments.py:233]: Get rid of this extra whitespace

**Review by andrewda — CHANGES_REQUESTED** [review]

File: cron/poll_read_issue_comments.py (hunk 2 above)
Comment (rudehn) [reply cron/poll_read_issue_comments.py:233]: Yeah i did earlier, but this fix is in several PRs and I don't want to have a merge conflict

Comment (andrewda) [reply cron/poll_read_issue_comments.py:233]: Alright 👍

**Review by andrewda — APPROVED** [review]

**Comment by rudehn** [comment]
Looks like this needs meritocracy approval now

**Review by phil-r — APPROVED** [review]

**Comment by PlasmaPower** [comment]
@phil-r please don't approve changes which break Travis

**Comment by rudehn** [comment]
I think it was that blank line that you fixed. I just left it in

**Comment by PlasmaPower** [comment]
From the diff it looks like you added the whitespace, not kept it.

**Comment by PlasmaPower** [comment]
Yep. Not sure if it's weird merge conflict resolution or what, but that change definitely came from this PR: https://github.com/chaosbot/Chaos/blame/4b65bd187f2892cc5ced5d2f60446606bbd73cb5/cron/poll_read_issue_comments.py#L233

**Comment by PlasmaPower** [comment]
To clarify, this PR kept one existing Travis error (good), but also added another Travis error (bad). I think you've got them confused.

**Pull request merged**